gym_minigrid/wrappers.py

```diff
@@ -156,25 +156,28 @@ class RGBImgObsWrapper(gym.core.ObservationWrapper):
     """
     Wrapper to use fully observable RGB image as observation,
     This can be used to have the agent to solve the gridworld in pixel space.
+    To use it, make the unwrapped environment with render_mode='rgb_array'.
     """
 
     def __init__(self, env, tile_size=8):
         super().__init__(env)
 
         self.tile_size = tile_size
 
-        self.observation_space.spaces['image'] = spaces.Box(
+        new_image_space = spaces.Box(
             low=0,
             high=255,
             shape=(self.env.width * tile_size, self.env.height * tile_size, 3),
             dtype='uint8'
         )
 
+        self.observation_space = spaces.Dict({**self.observation_space, 'image':new_image_space})
```

Comment:
What is the type of `self.observation_shape`? If it is a `Dict` then I'm not sure about `**` without `.spaces`.
I hope that makes sense, I just want to check that it is correct

Reply:
Good point ! It works, but I don't know why it works. It's safer to add `.spaces`. I'll do this in another commit. Thanks.